tests/README.md

```diff
@@ -7,12 +7,16 @@ To run the tests, you can either use the CI, or run the tests locally.
 
 ## Running the tests locally
 
-1. Run `cargo build --release -p chariott -p kv-app` to build the required projects.
-2. Run Chariott and the Key-Value app in the background, using
-`./target/release/chariott &` and `./target/release/kv-app &` (assuming you are
-in the repository root). If you want to display debug logs, make sure to set the
-log level via the `RUST_LOG` environment variable.
-3. Execute the tests using `cargo test --test "*e2e"`.
+Assuming the current working directory is the project root, run:
+
+```sh
+cargo build --release -p chariott -p kv-app
+CHARIOTT_REGISTRY_TTL_SECS=2 ./target/release/chariott &
```

Comment:
How frequently does the key-value application re-announce itself? If it's longer than two seconds, Chariott will evict the key-value application from the registry and it will reregister, causing unreliable E2E tests.

Reply:
Good point, but funny this didn't break locally unless I had my setup wrong. Anyway, Fixed with 28248bc741f87d102e5e76457fa35600d93233d7.